stan/math/rev/mat/fun/matrix_exp_action_handler.hpp

```diff
@@ -0,0 +1,117 @@
+#ifndef STAN_MATH_REV_MAT_FUN_MATRIX_EXP_ACTION_HANDLER_HPP
+#define STAN_MATH_REV_MAT_FUN_MATRIX_EXP_ACTION_HANDLER_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <stan/math/rev/core.hpp>
+#include <vector>
+
+double l1norm(const Eigen::MatrixXd& m) {
+  return m.colwise().lpNorm<1>().maxCoeff();
+}
+
```

Comment:
This is actually a neat feature. I'd put this in its own file for usage in other functions. Also add a brief doc describing what the function does.

Reply:
Fair enough. Let's roll with that for now.